packages/ui/src/components/BenefitCard/BenefitCard.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { ReactElement } from 'react';
+import { Box, BoxProps } from '@chakra-ui/react';
+
+import { Typography } from '../Typography';
+
+export interface BenefitCardProps extends BoxProps {
+  icon: ReactElement;
+  title?: string;
+  subtitle?: string;
+}
+
+export const BenefitCard = ({ icon, title, subtitle, ...props }: BenefitCardProps) => {
+  return (
+    <Box
+      bg="gray.50"
+      borderRadius="8px"
+      p="32px"
```

Comment:
🤔 Space between content and the icon should equal 24px. Left and right paddings are also 24px instead of 32px.